functions/completed-checks/index.js

```diff
@@ -79,6 +79,25 @@ async function savePayloadToAdminDatabase (completedCheckMessage, logger) {
     throw error
   }
 
+  // Don't process any checks more than once
+  if (check.receivedByServerAt) {
+    const msg = `completed-check: ERROR: payload re-submission is banned for check ${checkData.checkCode}`
+    logger.error(msg)
+    throw new Error(msg)
+  }
+
+
+  if (!checkData.startedAt) {

```

Comment:
@jon-shipley under which circumstances the `startedAt` field would not be populated?

Reply:
is it only in case check-started function fails?

Reply:
@GeorgeChatzigiannis 

1.  The client browser had networking issues
2. The completed check message was received before the check started message (busy system?)

Reply:
Ok that makes sense to me, thanks.